packages/google-pay-integration/e2e/GooglePayBraintreeInPaymentStep.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { PaymentStepAsGuestPreset, test } from '@bigcommerce/checkout/test-framework';
+
+import {
+    afterSignIn,
+    braintreePayPal,
+    braintreePayPalCredit,
+    googlePay,
+    internalOrder,
+    order222,
+    orderPayment,
+    payment,
+} from './GooglePayBraintreeMockingResponses';
+
+test.describe('Sample Test Group', () => {
```

Comment:
Well, it's not a sample anymore. 😂

Reply:
Thanks! Test description has been renamed)